fvm/src/gas/price_list.rs

```diff
@@ -115,6 +115,11 @@ lazy_static! {
             }
         },
```

Comment:
can we put `sig_cost` and `on_verify_signature` behind the feature-flag as well?

Reply:
Fixed.